### ISSUE TITLE
Some glosses have 'None' as value of one of the Phonology fields.

### ISSUE THREAD

**Issue by vanlummelhuizen** [Signbank/Global-signbank#328]
The problematic glosses are:

- CASSIS, 2134, field: relOriMov
- DOORGEVEN-C, 705, field: oriCh
- ST-MICHIELSGESTEL, 273, field: movCh
- PERSOON-A.PL, 658, field: virtObj (this is actually a char field, so not necessarily a problem)
- wesseltest3, 3117, field: handedness (this is a test gloss of course)

So far none of these caused errors, but with some of the new code by @susanodd they did. 

@ocrasborn how shall we solve this/who will solve this?

**Comment by susanodd** [comment]
To clarify, on the phonology fields with a choice, "no choice" is 0 (zero), and "never filled in" is a null value. But somehow the value 'None' appears in the database records mentioned above.
This leads to a type error on the query that searches for Minimal Pairs / Homonyms. The query looks for "empty" phonology fields and chokes on 'None' in the where comparison (the query crashes... because...)
Either the field being compared is not a string (but 'None' is a string, so it's not an empty value) or not an integer (the index of a pulldown choice is a number,where 0 is "no choice") (here, if None is a string, not one of the indices, it can't be compared to numbers. But if the query is written as relOriMov = None, the database looks for a field named None, which is a syntax error.)

**Comment by ocrasborn** [comment]
I have no clue where it came from! It's never been one of the possible values we created for most of the fields. And in fact, when I edit the signs now, the pull-down menu automatically changes to '-', which is what it should be. I've thus edited and saved all of them. For the char field virtObj, I've simply removed the content altogether, it wasn't necessary to have it there, but of course it could arise again in the future because users are free to type in anything.
